bus-mapping/src/evm.rs

```diff
@@ -192,6 +198,25 @@ impl EvmWord {
     pub fn to_hex(self) -> String {
         hex::encode(self.to_be_bytes())
     }
+
+    /// Returns an added `EvmWord`
+    pub fn adc(self, number: EvmWord) -> Result<EvmWord, Error> {
```

Comment:
As it can be seen in `Exceptions` section: https://github.com/appliedzkp/zkevm-specs/blob/master/specs/opcode/01ADD_03SUB.md#exceptions

There's no error due to overflow. Just due to stack underflow. THerefore that would need addressing.
I also agree with the comments that @ed255 added.

Reply:
Thank you!
I changed.